lib/orogen/templates/proxies/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Generated from orogen/lib/orogen/templates/proxies/CMakeLists.txt
+
+
+SET(PROXYLIB_NAME <%= component.name %>-proxies)
```

Comment:
I'm trying to (slowly, but surely) move away from using 'component' as the project-holding variable. Use 'project' instead.